packages/pages/src/dev/server/server.ts

```diff
@@ -54,6 +55,11 @@ export const createServer = async (
     ));
   }
 
+  // Will use defaultDevServerPort if available, otherwise fall back to a random available port
```

Comment:
Is it possible to choose the next sequential port? For instance if 5173 is occupied we pick 5174 and so on. I looked through the docs for get-port and didn't see a flag for this, if it's annoying it's not super important.

Reply:
updated! will use the next sequential port from 5173 to 6000 (have to pick the last port of the range.), otherwise, it will fallback to a random port.